rust/otap-dataflow/crates/otlp/src/fake_signal_receiver/config.rs

```diff
@@ -23,25 +23,52 @@ pub enum OTLPSignal {
 /// Configuration should take a scenario to play out
 #[derive(Clone, Deserialize, Serialize)]
 pub struct Config {
-    // A ordered list of steps defining various signals to emit
-    steps: Vec<ScenarioStep>,
+    // Configuration of the traffic to generate
+    traffic_config: TrafficConfig,
 
     resolved_registry: ResolvedRegistry,
```

Comment:
We should use a VirtualDirectoryPath here and then call the weaver resolver to get a `ResolvedRegistry`.